exec/java-exec/src/main/java/org/apache/drill/exec/store/sys/SystemTable.java

```diff
@@ -31,28 +31,30 @@
  * </p>
  */
 public enum SystemTable {
-  OPTION("options", false, OptionValueWrapper.class) {
+  OPTION_OLD("options_old", false, OptionValueWrapper.class) {
```

Comment:
Maybe rename as `OPTIONS_OLD` and `OPTIONS` for consistency? (making it plural).

Reply:
Done.